group2

```diff
@@ -0,0 +1,8 @@
+igor 
+lucy
+kamtin
+steph
```

Comment:
Hey Igor  my prefer name is Christelle not Stephanie thanks.

Reply:
ok let me correct it